02_classify_stratified/nbconverted/plot_purity_regression_results.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Plot tumor purity prediction results
+# 
+# Here, we'll visualize the results of our tumor purity prediction. We're predicting real-valued purity labels here (e.g. elastic net regression).
+# 
+# Results analyzed here are generated by the `run_purity_prediction.py` script, without the `--classify` argument.
```

Comment:
So by default it runs regression, unless you specify `--classify`?

Reply:
Yep, exactly. I guess my thought there was that the natural way to think about predicting continuous purity values was using regression, so I decided to make that the default and add a flag to binarize the output.